Source/Upstream: Yes, fixed in git master
Reason: Fix build with ffmpeg 8

From f852d815a7b91b024b9b0d162c3ef2ebc8133200 Mon Sep 17 00:00:00 2001
From: Paul Brossier <piem@piem.org>
Date: Sat, 15 Nov 2025 11:44:50 +0100
Subject: [PATCH] [source_avcodec] use #elif for conciseness, thanks to @bkuhls
 (closes gh-427)

---
 src/io/source_avcodec.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/io/source_avcodec.c b/src/io/source_avcodec.c
index 1fb14bf51..5ebaff165 100644
--- a/src/io/source_avcodec.c
+++ b/src/io/source_avcodec.c
@@ -66,13 +66,11 @@
 
 #if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(56, 56, 0)
 #define AUBIO_AVCODEC_MAX_BUFFER_SIZE FF_MIN_BUFFER_SIZE
-#else
-#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(60, 40, 100)
+#elif LIBAVCODEC_VERSION_INT < AV_VERSION_INT(60, 40, 100)
 #define AUBIO_AVCODEC_MAX_BUFFER_SIZE AV_INPUT_BUFFER_MIN_SIZE
 #else
 #define AUBIO_AVCODEC_MAX_BUFFER_SIZE 16384
 #endif
-#endif
 
 #if LIBAVCODEC_VERSION_MAJOR >= 59
 #define FF_API_LAVF_AVCTX 1
